apps/invite/urls.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.routers import DefaultRouter
+
+from .views import InvitationViewSet
+
+router = DefaultRouter()
+
+router.register(r"invitations", InvitationViewSet, basename="invitation")
```

Comment:
This will results in the url being `invitation/invitation/` so the url here can be `""`